web/src/components/dashboard/HideableTasks.test.tsx

```diff
@@ -28,7 +28,7 @@ describe("<HideableTasks />", () => {
     render(
       <WithStatefulUserData initialUserData={generateUserDataForBusiness(business)}>
         <HideableTasks />
-      </WithStatefulUserData>
+      </WithStatefulUserData>,
```

Comment:
I don't love trailing commas in the TSX render. Any way we can modify prettier to not do this?